app/responses.py

```diff
@@ -74,7 +75,7 @@ def jsonencoder_lite(obj):
     encoding large lists. This encoder only encodes the bare necessities
     needed to work with serializers like ORJSON.
     """
-    if isinstance(obj, decimal.Decimal):
+    if isinstance(obj, decimal.Decimal) or isinstance(obj, asyncpg.pgproto.pgproto.UUID):
```

Comment:
Not sure why this change is showing up in the merge to develop, since this change is definitely already in develop.  Maybe you need to update your change (I see if it showing at least once conflict)?